lib/phoenix/channel/server.ex

```diff
@@ -181,4 +203,40 @@ defmodule Phoenix.Channel.Server do
     got #{inspect result}
     """
   end
+
+  ## Handle replies
+
+  defp handle_reply(socket, {status, payload}, :handle_in)
```

Comment:
We were not verifying replies on `:stop`, only on `:reply`, so I created this function.